README.md

```diff
@@ -23,6 +23,11 @@ This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-gov
 
 EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently there is a lot of manual labour that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
 
+- [Project Goals](#project-goals)
+- [How to Help](#how-to-help)
+- [Project Overview](#project-overview)
+- [License & Copyright](#license-copyright)
```

Comment:
This link isn’t working—I think it needs to be `#license--copyright` (double dash because of the `&` in the title).